Website/src/rendering/src/components/SessionInformation.tsx

```diff
@@ -47,134 +63,132 @@ const SessionInformation = (props: SessionInformationProps): JSX.Element => {
-              <DateField
-                tag="span"
-                field={props.fields.Date}
-                render={(date) =>
-                  'Time: ' +
-                  date?.toLocaleTimeString('en-US', {
-                    hour: '2-digit',
-                    minute: '2-digit',
-                  })
-                }
-              />
+              <Image field={props.fields?.Image} alt={props.fields?.Name?.value} />
+              {props.fields.Rooms && props.fields.Rooms.length > 0 && (
+                <div className="info-text">
+                  <span>
+                    <FontAwesomeIcon className="icon" icon={faDoorOpen} />
+                  </span>
+                  <Text tag="p" field={props.fields.Rooms[0].fields.Name}></Text>
+                </div>
+              )}
+              {props.fields.Day &&
+                props.fields.Day.length > 0 &&
+                props.fields.Day.map((day, index) => (
+                  <div key={index} className="info-text">
+                    <span>
+                      <FontAwesomeIcon className="icon" icon={faCalendar} />
+                    </span>
+                    <Text tag="p" field={day.fields.Name}></Text>
+                  </div>
+                ))}
+              {props.fields.Timeslots && props.fields.Timeslots.length > 0 && (
+                <div className="info-text">
+                  <span>
+                    <FontAwesomeIcon className="icon" icon={faClock} />
+                  </span>
+                  <span>{getSessionTime(props.fields.Timeslots)}</span>
+                </div>
+              )}
             </div>
-
-            <div className="hidden md:block pt-4" title="TODO: hardcoded for now...">
-              <div className="font-bold">Related Sessions</div>
-              <div className="border border-gray p-5 my-5">
+            <div className="related-sessions" title="TODO: hardcoded for now...">
+              <div className="session-title">Related Sessions</div>
+              <div className="session">
                 <p>Mon, 19th | 9:00 AM</p>
-                <p className="font-bold">10 Tips to get the most out of your routines</p>
+                <p className="session-title">10 Tips to get the most out of your routines</p>
               </div>
             </div>
           </div>
-          <div className="col-span-1 md:col-span-3 space-y-5">
-            <Text tag="span" className="btn--main bg-yellow" field={props.fields.Type}></Text>
-            <Text
-              tag="h2"
-              className="text-2xl md:text-3xl font-extrabold text-blue"
-              field={props.fields.Name}
-            ></Text>
-            <div className="grid grid-cols-1 lg:grid-cols-2 gap-4">
-              <div className="font-bold col-span-1 lg:col-span-2">{speakerHeader}</div>
+          <div className="description-col">
+            <div>
+              <Text tag="span" className="eyebrow" field={props.fields.Type}></Text>
```

Comment:
Why are the eyebrow `<span>` here and `<div>` in `SpeakerInformation.tsx`? Shouldn't it be identical everywhere?